src/bin/flamegraph.rs

```diff
@@ -116,7 +116,14 @@ struct Opt {
     /// If you set this flag you need to be sure your
     /// input stack lines are already sorted.
     #[structopt(long = "no-sort")]
-    pub no_sort: bool,
+    no_sort: bool,
+
+    /// Generate stack-reversed flame graph.
+    /// Note that stack lines must always be sorted
+    /// after reversing the stacks so the no-sort
```

Comment:
```suggestion
    /// after reversing the stacks so the `no_sort`
```

Reply:
This gets printed in the help text of the CLI, and the name of the option to the user is `no-sort` with a dash, not an underscore.

Reply:
Ah, yes, you're right, I missed that this is in the CLI.